pybamm/util.py

```diff
@@ -364,23 +365,50 @@ def is_jax_compatible():
     )
 
 
-def install_jax():  # pragma: no cover
-    """Install jax, jaxlib"""
+def install_jax(arguments=None):  # pragma: no cover
+    """
+    Install compatible versions of jax, jaxlib.
+
+    Command Line Interface:
+    -----------------------
+    >>> pybamm_install_jax
+
+    optional arguments:
+    -h, --help   show help message
+    -f, --force  force install compatible versions of jax and jaxlib
+    """
+    parser = argparse.ArgumentParser(description="Install jax and jaxlib")
+    parser.add_argument(
+        "-f",
+        "--force",
+        action="store_true",
+        help=f"force install compatible versions of jax ({JAX_VERSION}) and jaxlib ({JAXLIB_VERSION})",  # noqa: E501
+    )
+
+    args = parser.parse_args(arguments)
+
     if system() == "Windows":
         raise NotImplementedError("Jax is not available on Windows")
+
+    # Raise an error if jax and jaxlib are already installed, but incompatible
+    # and --force is not set
     elif importlib.util.find_spec("jax") is not None:
-        if not is_jax_compatible():
+        if not args.force and not is_jax_compatible():
             raise ValueError(
-                "Jax is already installed but the installed version of jax or jaxlib is not supported by PyBaMM",  # noqa: E501
+                f"""
+Jax is already installed but the installed version of jax or jaxlib is not supported by PyBaMM.
+
+You can force install compatible versions of jax ({JAX_VERSION}) and jaxlib ({JAXLIB_VERSION}) using the following command:
+pybamm_install_jax --force""",  # noqa: E501
```

Comment:
Can you use single quotes here instead, like https://github.com/pybamm-team/PyBaMM/blob/fb6ebaa12900d227d3a247afdb7acfa9f5b7c62d/pybamm/simulation.py#L1000?